.github/workflows/run-e2e-tests-saas.yaml

```
@@ -0,0 +1,30 @@
+name: Cypress E2E SaaS Tests
+on: push
+jobs:
+  cypress-run:
+    # If you want this job to run on your fork, remove the below "if" line.
+    # You will need to "fix" the tests as they are specific to Adobe Commmerce's
+    # demo backend.
+    if: github.repository == 'hlxsites/aem-boilerplate-commerce'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install root dependencies
+        run: npm ci
+      - name: Start server in the background
+        run: npm run cypress:saas &
```

Instead of this, just:
```suggestion
        run: aem up --url https://main--boilerplate-accs--adobe-commerce.aem.page
```

---

And in `run-e2e-tests.yaml` for paas, you would do above but with paas url.

---

@sirugh This is not working, what i'm missing here 😕 

`Run aem up --url https://main--boilerplate-paas--adobe-commerce.aem.live &
  aem up --url https://main--boilerplate-paas--adobe-commerce.aem.live &
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/7417fc01-0[3](https://github.com/hlxsites/aem-boilerplate-commerce/actions/runs/14988759355/job/42107571549?pr=445#step:4:3)1a-4495-9ea8-e38c4a83c1c9.sh: line 1: aem: command not found`

---

`aem command not found`.

Do you install aem in this env? We install as dev dep: https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/package.json#L26.

Does the CLI env install dev dependencies?

---

@sirugh I dont have strict NODE_ENV=production set on CI, so it should install AFAIK. But, even on my local this only works if I install it globally `npm install -g @adobe/aem-cli ` .  We should not be doing `-g` on ci.

---

As discussed, appears the only way to install the aem-cli is via `-g` (based on their readme). So it is what it is. I wouldn't be too worried though, because presumably the git runners are isolated sandboxes that get wiped afterwards.